docs/en/ingest-management/elastic-agent/configuration/autodiscovery/kubernetes-hints-autodiscover.asciidoc

```diff
@@ -168,6 +171,143 @@ After deploying this Pod, the data will start flowing in automatically. You can
 NOTE: All assets (dashboards, ingest pipelines, and so on) related to the Redis integration are not installed. You need to explicitly <<install-uninstall-integration-assets,install them through {kib}>>.
 
 
+[discrete]
+=== Hints autodiscovery for kubernetes log collection
+
+The log collection for Kubernetes autodiscovered pods can be supported by using  https://github.com/elastic/elastic-agent/tree/main/deploy/kubernetes/elastic-agent-standalone/templates.d/container_logs.yml[container_logs.yml template]. Users need to emit a caontainer_logs mapping so as to start collecting logs for all the discovered containers *even if no annotations are present in the containers*. 
+
+Enabling hints in the kubernetes standalone manifest: 
+[source,yaml]
+----
+providers.kubernetes:
+  node: ${NODE_NAME}
+  scope: node
+  hints:
+    enabled: true
+----
+
+Example to manually mount the needed container_logs template under the inputs.d folder of Elastic Agent
```

Comment:
Why we need this extra mount? All templates are mounted automatically under `/etc/elastic-agent/inputs.d` if we uncomment the respective parts in the k8s manifests. So I see no need to specifically mount this one.

Reply:
I think that if we want an example with custom template's addition we should create a separate section for this and be specific there.

Reply:
Ok removed mount info and updated the section a bit